_seattle/API Chaya.md

```diff
@@ -1,18 +1,18 @@
 ---
 name: API Chaya
-address: API Chaya, P.O. Box 14047, Seattle, WA 98114
+address: 
 lat: 
```

Comment:
Hmm ok. We should probably decide to put it somewhere on the map though? Otherwise it won't show up on the front-end at all. I suppose we could just put in the lat/long for "Seattle, WA 98114"?